### PR TITLE
getGrantPayee grant storage

### PR DESCRIPTION
**Pull request by mk1r**
small PR - no need for storage modifier inside getGrantPayee function, also added gas cost test for getGrantPayee

branched off #497

### CODE REVIEW

**Review by mds1 — APPROVED** [review]
LGTM! Changed the target branch for now to make the diff easier to review. Let's wait for #497 to get merged before changing the target branch back then merging this one

**Review by apbendi — APPROVED** [review]
lgtm!

**Pull request merged**
